src/shared/dlt_offline_trace.c

```diff
@@ -339,6 +341,22 @@ int dlt_offline_trace_delete_oldest_file(DltOfflineTrace *trace) {
 
 DltReturnValue dlt_offline_trace_check_size(DltOfflineTrace *trace) {
 
+    struct stat status;
+
+    /* check for existence of offline trace directory */
+    if(stat(trace->directory, &status) == -1)
```

Comment:
I would prefer to have a space after if:  `if (...)`

I know there is currently no aligned coding style and (not yet) a guideline available, but I like to improve that.